### PR TITLE
Add mark all test

### PR DESCRIPTION
**Pull request by samccone**
tests :dancer: :skull: :fire:

### CODE REVIEW

File: examples/spine/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+ <!doctype html>
```
Comment (boushley) [inline examples/spine/index.html]: Any reason these spaces are being added?

Comment (samccone) [inline examples/spine/index.html]: they were added to trip CI to run the test suite over these files :) (I will drop the commit before I manually merge)

Our CI now is really smart and will only run the integration tests over the examples that have been changed, however when you make changes to the test files... no tests run so you need to manually  touch some files in projects (it is a bit of a hack... however I have not had the time / motivation to make it less of a hack ((any help welcome here)))

Comment (boushley) [inline examples/spine/index.html]: Makes sense. I'll see if I can find some time to dig into that.

Comment (samccone) [inline examples/spine/index.html]: no worries any help is more than appreciated :smile:

File: tests/test.js [hunk 2]
```diff
@@ -148,6 +148,16 @@ module.exports.todoMVCTest = function (frameworkName, baseUrl, speedMode, laxMod
...
 
+			test.it('should correctly update the complete all checked state', function () {
+				//manually check all items
+				page.toggleItemAtIndex(0);
+				page.toggleItemAtIndex(1);
+				page.toggleItemAtIndex(2);
+
+				//ensure checkall is in the correct state
+				testOps.assertCompleteAllIsChecked();
+			});
```
Comment (passy) [inline tests/test.js]: Thank youu!

File: tests/test.js [hunk 3]
```diff
@@ -148,6 +148,16 @@ module.exports.todoMVCTest = function (frameworkName, baseUrl, speedMode, laxMod
 				testOps.assertItemAtIndexIsCompleted(2);
 			});
 
+			test.it('should correctly update the complete all checked state', function () {
+				//manually check all items
```
Comment (passy) [inline tests/test.js]: Super nitty: The other comments in the file are introduced with `//` + space. :)

**Comment by passy** [comment]
:+1:

File: tests/test.js (hunk 3 above)
Comment (samccone) [inline tests/test.js]: kk

**Comment by samccone** [comment]
fyi tests are borked ATM due to...

either selenium pushed a new breaking change
chromium nightly broke something
new chrome driver broke something
travis changed how headless browsers work... breaking the world :earth_asia: 

:crying_cat_face:

**Comment by samccone** [comment]
manually landed

**Pull request closed without merging**